reference-implementation/unittests/tosa.cpp

```diff
@@ -80,6 +80,59 @@ TEST(tosa, clamp) {
 
     EXPECT_THAT(result, Pointwise(FloatEq(), expected_result));
   }
+
+  // These are the tests of the former ReluN operation, which has been removed
+  // from the TOSA specification. clamp with min_value=0 can be used instead.
+  {
+    Tensor0D<int32_t> operand{0};
+    int32_t max_value = 0;
+    Tensor0D<int32_t> expected_result{0};
+    Tensor0D<int32_t> result = tosa::clamp(operand, {0}, max_value);
+
+    EXPECT_THAT(result, Pointwise(Eq(), expected_result));
+  }  {
```

Comment:
This needs to be formatted!

Reply:
Yep.. the linter action was broken and thus I didn't noticed before. This is fixed now (with the linter action revised in the background).